### ISSUE TITLE
Failed to run scripts on Nightly with message alert.bind(...).apply is not a function

### ISSUE THREAD

**Issue by tiansh** [greasemonkey/greasemonkey#2345]
```javascript
// ==UserScript==
// @name        Example Script
// @namespace   -
// @include     *
// @version     1
// @grant       GM_getValue
// ==/UserScript==

try {
  alert.bind(window).apply(null, ['hello']);
} catch (e) { alert(e); }
```

46.0a1 (2015-12-28) this issue keeps about a week
GM3.6 and 3.7b1

grant none will make it works again, so l believe it is due to the sandbox.

**Comment by janekptacijarabaci** [comment]
Confirmed on:
Firefox 45.0a1 Nightly (2015-12-09, Build ID 20151209095500)

Not confirmed on:
Firefox 45.0a1 Nightly (2015-12-07, Build ID 20151207030210)

See also #2129
(https://bugzilla.mozilla.org/show_bug.cgi?id=1163019 - Modified: 2015-06-02 09:27 PDT :-/)

**Comment by janekptacijarabaci** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1237177

**Comment by janekptacijarabaci** [comment]
Resolved fixed (Firefox Nightly 46.0a1 (2016-01-12)+)
